pkg/services/publicdashboards/service/service.go

```diff
@@ -142,11 +142,6 @@ func (pd *PublicDashboardServiceImpl) Create(ctx context.Context, u *user.Signed
 		return nil, err
 	}
 
-	// set default value for time settings
-	if dto.PublicDashboard.TimeSettings == nil {
-		dto.PublicDashboard.TimeSettings = &TimeSettings{}
-	}
-
 	// validate fields
 	err = validation.ValidatePublicDashboard(dto, dashboard)
```

Comment:
wonder if its worth validating the `share` field in here? If the email sharing feature isn't enabled, the only acceptable value is `public`

Reply:
I'm not sure. Should we spread through the codebase the feature flag for everything? The feature would not work even if you update a dashboard with share `email` because the endpoints are behind a feature flag.

Reply:
okay, that should be fine then